reports.go

```diff
@@ -92,16 +187,830 @@ func (c *Client) CreateReport(reqPayload *CreateReportRequest) (*Report, error)
+
+	req, err := http.NewRequest(http.MethodGet, u.String(), nil)
+	if err != nil {
+		return nil, NewError([]int{
+			http.StatusOK,
+		}, nil)
+	}
+	req.Header.Add("Content-Type", "application/json")
+	req.SetBasicAuth(c.APIKey, "")
+
+	resp, err := c.client.Do(req)
+	if err != nil {
+		return nil, NewError([]int{
+			http.StatusOK,
+		}, resp)
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		return nil, NewError([]int{
+			http.StatusOK,
+		}, resp)
+	}
+	defer func() {
+		if resp.Body != nil {
+			resp.Body.Close()
+		}
+	}()
+
+	b, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return nil, NewError([]int{
+			http.StatusOK,
+		}, resp)
+	}
+
+	getResp := &EmploymentVerification{}
+	err = json.Unmarshal(b, getResp)
+	if err != nil {
+		return nil, NewError([]int{
+			http.StatusOK,
+		}, resp)
+	}
+
+	return getResp, nil
+}
+
+const identityDocumentVerificationSearchPath = "/v1/identity_document_verifications"
+
+func (r *Report) GetIdentityDocumentSearch(identityDocumentVerificationID string, c *Client) (*IdentityDocumentVerification, Error) {
+
+	if identityDocumentVerificationID == "" {
+		return nil, NewError([]int{
+			http.StatusOK,
+		}, nil)
+	}
+
+	rel, err := url.Parse(identityDocumentVerificationSearchPath)
+	if err != nil {
+		return nil, NewError([]int{
+			http.StatusOK,
+		}, nil)
+	}
+
+	u := *c.BaseURL
+	u.Path = path.Join(u.Path, rel.String(), identityDocumentVerificationID)
+
+	req, err := http.NewRequest(http.MethodGet, u.String(), nil)
+	if err != nil {
+		return nil, NewError([]int{
+			http.StatusOK,
+		}, nil)
+	}
+	req.Header.Add("Content-Type", "application/json")
+	req.SetBasicAuth(c.APIKey, "")
+
+	resp, err := c.client.Do(req)
+	if err != nil {
+		return nil, NewError([]int{
+			http.StatusOK,
+		}, resp)
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		return nil, NewError([]int{
+			http.StatusOK,
+		}, resp)
+	}
+	defer func() {
+		if resp.Body != nil {
+			resp.Body.Close()
+		}
+	}()
+
+	b, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return nil, NewError([]int{
+			http.StatusOK,
+		}, resp)
+	}
+
+	getResp := &IdentityDocumentVerification{}
+	err = json.Unmarshal(b, getResp)
+	if err != nil {
+		return nil, NewError([]int{
+			http.StatusOK,
+		}, resp)
+	}
+
+	return getResp, nil
+}
+
+// GetScreenings returns all the screenings for a report
+func (r *Report) GetScreenings(c *Client) (*Screenings, error) {
+	cs := &Screenings{}
+	var err Error
+	var wg sync.WaitGroup
+
+	screenIds := make(map[ScreenType]string)
+	screenIds[SsnTrace] = r.SSNTraceID
+	screenIds[Sexoffendersearch] = r.SexOffenderSearchID
+	screenIds[GlobalWatchListsearch] = r.GlobalWatchlistSearchID
+	screenIds[NationalCriminalsearch] = r.NationalCriminalSearchID
+	screenIds[FederalCriminalsearch] = r.FederalCrimeSearchID
+	screenIds[MotorVehiclereport] = r.MotorVehicleReportID
+	if len(r.CountyCriminalSearchIDs) > 0 {
+		screenIds[CountryCriminalsearch] = r.CountyCriminalSearchIDs[0]
```

Comment:
You're only getting the first element here when we should be getting everything.